samples/junit-tests/junit_tests_specs.json

```diff
@@ -727,6 +727,38 @@
         "/reserved_keywords": {
             "get": {
                 "operationId": "get_reserved_keywords",
+                "parameters": [
+                    {
+                        "in": "query",
+                        "name": "class",
```

Comment:
Should we add one entry for `data`?

Reply:
I did copy the reserved keywords already defined in the `reserved_keywords` model.
We can add `data` as well but it would provide no additional info as it is defined in `KotlinLangUtils.kotlinReservedWords` as `class` and the others. But up to you ... not opposed to it ;)

https://github.com/Yelp/swagger-gradle-codegen/blob/8b24632314d09499a9a27059e68b23b80ece2ec5/gradle-plugin/plugin/src/main/java/com/yelp/codegen/utils/KotlinLangUtils.kt#L26-L39